### PR TITLE
docs: update the README to state that this plugin support fastify 4

### PR DESCRIPTION
**Pull request by rluvaton**
#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [x] documentation is changed or added
- [ ] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,10 +4,14 @@
...
-Plugin for serving static files as fast as possible. Supports Fastify version `3.x`.
-
-Please refer to [this branch](https://github.com/fastify/fastify-static/tree/2.x) and related versions for Fastify `^2.0.0` compatibility.
-Please refer to [this branch](https://github.com/fastify/fastify-static/tree/1.x) and related versions for Fastify `^1.11.0` compatibility.
+Plugin for serving static files as fast as possible. Supports Fastify version `4.x`.
+
+| Fastify version | branch                                                               |
+| --------------- | -------------------------------------------------------------------- |
+| `^4.x`          | This branch                                                          |
+| `^3.x`          | [CHANGE ME](https://github.com/fastify/fastify-static/tree/CHANGEME) |
```
Comment (rluvaton) [inline README.md:12]: There is no branch for supporting `3.x` versions, could you please create and I'll update this accordinly

Comment (climba03003) [reply README.md:12]: https://github.com/fastify/fastify-static/tree/v5.x

**Comment by rluvaton** [comment]
@climba03003 done, thanks

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**